Implementation/service/Admin.py

```diff
@@ -0,0 +1,80 @@
+from data.Database import connect_to_db
+from core.User import User
+from core.Cafeteria import MenuItem
+from service.Validation import Validation
+from notification.notification import add_notification
+from core.Exception import DatabaseConnectionError, InvalidInputError, ItemNotFoundError, NotificationError
+
+class Admin(User):
+    def __init__(self, user_id, user_name):
+        super().__init__(user_id, user_name, 'Admin')
+
+    def add_menu_item(self, name, price, availability, food_type, spice_level, is_sweet):
+        item = MenuItem(None, name, price, availability, food_type, spice_level, is_sweet)
```

Comment:
Look at common things across different methods and check if they can be put together in a function and reused.